drivers/spi/spi_ll_stm32.c

```diff
@@ -1761,6 +1765,7 @@ static int spi_stm32_init(const struct device *dev)
 			   (.midi_clocks = DT_INST_PROP(id, midi_clock),))	\
 		IF_ENABLED(DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi),		\
 			   (.mssi_clocks = DT_INST_PROP(id, mssi_clock),))	\
+		.peripheral_use_soft_nss = DT_INST_PROP(id,periph_use_soft_nss) \
```

Comment:
As requested by CI + add trailing comma:
```suggestion
		.peripheral_use_soft_nss = DT_INST_PROP(id, periph_use_soft_nss), \
```